frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -80,7 +80,7 @@ export type LayerPopupInformation = PopupContentConfig & {
 };
 
 const defaultFilterValues: IPropertyFilter = {
-  searchBy: 'name',
+  searchBy: 'address',
```

Comment:
Is it supposed to be default to `address`?

Reply:
Only because I had to move name off that `PropertyOptions `and into its own input